frontend/src/containers/Dashboard/actions.js

```diff
@@ -1,16 +1,42 @@
-export const getHouseIssues = () => {
-  return new Promise( (resolve, reject) => {
-    resolve(
-      [
-        {
-          _id: 1,
-          house_image: "//via.placeholder.com/200x200",
-          issues: [
-            {_id: 1, status: "open", title: "test issue", description: "This is a test issue"},
-            {_id: 2, status: "resolved", title: "kitchen tap leaking", description: "can't turn the kitchen tap off!"}
-          ]
-        }
-      ]
-    );
-  });
-};
+export const HOUSE_ISSUES_HAS_ERRORED = "HOUSE_ISSUES_HAS_ERRORED";
+export const HOUSE_ISSUES_IS_LOADING = "HOUSE_ISSUES_IS_LOADING";
+export const HOUSE_ISSUES_FETCH_DATA_SUCCESS = "HOUSE_ISSUES_FETCH_DATA_SUCCESS";
+
+export function houseIssuesHasErrored(bool) {
+  return {
+    type: HOUSE_ISSUES_HAS_ERRORED,
+    hasErrored: bool
```

Comment:
this and the next action confuses me. Do we need to do hasErrored: bool. and have the parameter called bool. I would suggest changing the parameter to `hasErrored` and then you can just remove `hasErrored: bool` this isn't useful and doesn't tell us anything (see third action). I mean yes JS is not statically typed but if anything you could add a blocker to say throw an error if anything other than a boolean is passed. But calling it a `bool` is a useless bit of naming. Or so I would see it.